source/slang/slang-emit-wgsl.cpp

```diff
@@ -343,6 +345,38 @@ static const char* getWgslImageFormat(IRTextureTypeBase* type)
     //
     ImageFormat imageFormat =
         type->hasFormat() ? (ImageFormat)type->getFormat() : ImageFormat::unknown;
+
+    if (imageFormat == ImageFormat::unknown)
```

Comment:
The `switch` statement right after this `if`, there is a case for `ImageFormat::unknown`.
This code change could be placed inside of the switch.

Reply:
I originally had it inside the switch statement. Having it outside makes it a bit easier to mentally parse imo. It's the difference between
```
handle edge cases
do main operation
```
and
```
do main operation except in the case of edge cases where you handle them
```